SetupDataPkg/Tools/ConfigEditor.py

```diff
@@ -968,6 +974,15 @@ def load_cfg_file(self, path, file_id, clear_config):
 
         self.config_xml_path = path
         self.output_current_status(f"{path} file is loaded")
+        # load xml file and get the hash value of all xml nodes
+        config_xml_hash = get_xml_full_hash(self.config_xml_path)
+
+        # compare the xml version with self.bios_git_info, 7 digits should be enough
+        if self.bios_schema_xml_hash != "Unknown" and config_xml_hash != self.bios_schema_xml_hash:

```

Comment:
Would "Unknown" represent a potential warning as well?

Reply:
thanks. i use None to represent the BIOS doesn't provide the hash variable.